kong/clustering.lua

```diff
@@ -661,6 +680,7 @@ function _M.handle_cp_websocket()
           ip = node_ip,
           version = node_version,
           sync_status = sync_status,
+          cert_exp_date = shared_cert_expiration_time
```

Comment:
nitpick: if we are returning the timestamp, we can call this `cert_expiry_timestamp` instead of "date", so it's more self-explantory. (also need to update migrations if we accept this comment).

Reply:
that is a good point, I've already named function `*_time` lets keep it consistent, thanks!